timedColorChange.js

```diff
@@ -13,17 +14,19 @@ function getRandomColor() {
 function setRandomColor() {
     var background = document.getElementsByTagName( 'main' )[0];
     var newColor = getRandomColor();
-
+    // checks if generated color has already been used
     if( used.includes( newColor ) ) {
```

Comment:
I think this checks only one time whether the randomly generated color has been used or not. What happens if the call inside the if statement returns another already-used color?